### PR TITLE
Edits to deployment docs

### PR DESCRIPTION
**Pull request by pierwill**
Edited overview, moved info on landing pages to landing_page.rst

## Status

Ready for review

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/deployment/landing_page.rst [hunk 1]
```diff
@@ -3,6 +3,19 @@
 Landing Page
 ============
 
+SecureDrop itself is located on a Tor hidden service. Organizations also need to
```
Comment (heartsucker) [inline docs/deployment/landing_page.rst]: I feel like a better way to phrase this is "SecureDrop runs as a Tor Hidden Service" or "SecureDrop is a Tor Hidden Service." Something about "is located on a" sounds off to my ear. But also I don't even know how to English any more, so additional input from the team would be helpful. :)

File: docs/deployment/landing_page.rst [hunk 2]
```diff
@@ -3,6 +3,19 @@
...
+create a SecureDrop *Landing Page* that will explain how SecureDrop works, give
+sources instructions on how to access the Tor hidden service, and disclose the
+risks. We also recommend including a privacy policy (see our :ref:`Sample
+Privacy Policy`) describing what data is collected and how it will be used by
+your organization.
+
+.. important:: Implementing SecureDrop will bring more attention to your
+               organization from security researchers, hackers, and others. A
```
Comment (b-meson) [inline docs/deployment/landing_page.rst]: do we want to include this scary reference to `hackers`. I don't want to scare monger potential news orgs. Additionally, the biggest risk is legal / national security isn't it?

Comment (pierwill) [reply docs/deployment/landing_page.rst]: Maybe just "security researchers and others"?

**Comment by pierwill** [comment]
force pushed suggest revisions

**Pull request merged**